src/utils/device.js

```diff
@@ -57,7 +79,27 @@ export function isAutomatedBolusDevice(manufacturer, pumpSettings = {}) {
  * @returns {Boolean}
  */
 export function isSettingsOverrideDevice(manufacturer, pumpSettings = {}) {
-  return manufacturer === 'tandem' && _.get(pumpSettings, 'deviceId', '').indexOf('tandemCIQ') === 0;
+  return (manufacturer === 'tandem' && _.get(pumpSettings, 'deviceId', '').indexOf('tandemCIQ') === 0)
+  || isLoop(pumpSettings);
+}
+
+/**
+ * Get the uppercased manufacturer name
+ * @param {String} manufacturer Manufacturer name
+ */
+export function getUppercasedManufacturer(manufacturer = '') {
+  return _.map(manufacturer.split(' '), part => (part === 'diy' ? _.upperCase(part) : _.upperFirst(part))).join(' ');
+}
+
+/**
+ * Get a list of standard settings overrides for a settings-overrideable device,
+ * with default fallbacks for missing keys
+ * @param {String} manufacturer Manufacturer name
+ * @returns {Array} settings overrides
+ */
+export function getSettingsOverrides(manufacturer) {
+  const overrides = _.cloneDeep(settingsOverrides);
```

Comment:
Ah, no. I basically copied the `getPumpVocaubary` function below, and followed it's pattern. It's not mutated, and I'll remove the cloning.
Re: `getPumpVocabulary`, I believe it was getting mutated somewhere, causing issues.  It gets passed around a lot, and I probably, at the time (6 yrs ago, so can't really recall), I opted to simply clone it rather than track down and fix the underlying issue.